### ISSUE TITLE
Html tag attributes do not keep

### ISSUE THREAD

**Issue by antoniopol06** [liferay/senna.js#130]
Base on this part of code
https://github.com/liferay/senna.js/blob/master/src/screen/HtmlScreen.js#L42-L52

Html tag attributes are not updated by innerHtml

**Comment by jbalsas** [comment]
This is particularly important if you navigate from a `ltr` site to a `rtl` one and vice-versa. In this case, you'll start with

```html
<html class="ltr" dir="ltr" lang="es-ES">
    [Left-to-Right content here in Spanish (es-ES)]
</html>
```

and then, if you switch and start serving your Right-to-Left content, you'll end up with:

```html
<html class="ltr" dir="ltr" lang="es-ES">
    [Right-to-Left content here in Hebrew (iw-IW)]
</html>
```

**Comment by eduardolundgren** [comment]
The only way here is to sync the attributes of the HTMLNode.

**Comment by brunobasto** [comment]
We might need to sync the attributes of the <body> node too. Not just the 'class' attribute. I'll work on that next.

**Comment by eduardolundgren** [comment]
Which node? HTMLNode?

**Comment by brunobasto** [comment]
Sorry, my previous comment was not markdown compatible :) (already fixed).

I was talking about the BodyNode.

**Comment by eduardolundgren** [comment]
SGTM. Both html/body needs to be synced. Anyone available to work on that?

**Comment by brunobasto** [comment]
Fixed on: af4fb6646349251ab25095a01f2e95df1782f041
